tests/scripts/all.sh

```diff
@@ -4802,7 +4802,7 @@ component_build_aes_via_padlock () {
     scripts/config.py unset MBEDTLS_AESNI_C
     scripts/config.py set MBEDTLS_PADLOCK_C
     scripts/config.py unset MBEDTLS_AES_USE_HARDWARE_ONLY
-    make CC=$ASAN_CC CFLAGS="$ASAN_CFLAGS -m32" LDFLAGS="-m32 $ASAN_CFLAGS"
+    make CC=gcc CFLAGS="$ASAN_CFLAGS -m32" LDFLAGS="-m32 $ASAN_CFLAGS"
```

Comment:
Perhaps add a comment explaining why we make an exception? Also, I'm curious to know that the "known issues with clang for this target" are.

Reply:
It's this kind of link error: https://github.com/NixOS/nixpkgs/issues/201199